crates/client/README.md

```diff
@@ -64,23 +64,23 @@ assert!(!a_data.is_empty());
 
 ## DNS-over-TLS and DNS-over-HTTPS
 
-DoT and DoH are supported. This is accomplished through the use of one of `native-tls`, `openssl`, or `rustls` (only `rustls` is currently supported for DoH).
+DoT and DoH are supported. This is accomplished through the use of `rustls`.
```

Comment:
I think we should get rid of this paragraph.

Reply:
Thanks updated